### PR TITLE
document the PodAndContainerStatsFromCRI feature

### PR DESCRIPTION
**Pull request by tzneal**
Fixes: #33509

### CODE REVIEW

File: content/en/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Potential improved performance if the container runtime already collects this information
+  during normal operations. In this case, the data can be re-used instead of being aggregated
+  again by the kubelet.
+
+- It further decouples the kubelet and the container runtime allowing collection of metrics for
+  container runtimes that don't run processes directly on the host with kubelet where they are
+  observable by cAdvisor (e.g. VM based runtimes).
+
+Collecting this information from the CRI will become the default and the current cAdvisor collection
+of metrics from the host will become fully deprecated in a future Kubernetes release.
```
Comment (sftim) [inline content/en/docs/reference/instrumentation/cri-pod-container-metrics.md:40]: Only make this commitment if it's one that SIG Architecture would 100% stand behind. If it's an ambition we have but we aren't promising to make it happen, avoid mentioning it here.

File: content/en/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+The benefits of relying on the container runtime for this information as opposed to direct
+collection with cAdvisor include:
+
+- Potential improved performance if the container runtime already collects this information
+  during normal operations. In this case, the data can be re-used instead of being aggregated
+  again by the kubelet.
+
+- It further decouples the kubelet and the container runtime allowing collection of metrics for
+  container runtimes that don't run processes directly on the host with kubelet where they are
+  observable by cAdvisor (e.g. VM based runtimes).
```
Comment (sftim) [inline content/en/docs/reference/instrumentation/cri-pod-container-metrics.md:37]:
```suggestion
  observable by cAdvisor (for example: container runtimes that use virtualization).
```

File: content/en/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+metrics via the {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI). You
+must enable the `PodAndContainerStatsFromCRI` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and
+use a compatible CRI implementation (containerd >= 1.6.0, CRI-O >= 1.23.0) to
+use the CRI based collection mechanism.
+
+<!-- body -->
+
+## CRI Pod & Container Metrics
+
+With `PodAndContainerStatsFromCRI` enabled, the kubelet will poll the underlying container
```
Comment (sftim) [inline content/en/docs/reference/instrumentation/cri-pod-container-metrics.md:26]:
```suggestion
With `PodAndContainerStatsFromCRI` enabled, the kubelet polls the underlying container
```

**Review by sftim — COMMENTED** [review]
Clearly an improvement!

LGTM for docs even with nits identified.

We should get a tech review - @tzneal can you recommend a reviewer for that?

File: content/en/docs/reference/instrumentation/cri-pod-container-metrics.md (hunk 1 above)
Comment (tzneal) [reply content/en/docs/reference/instrumentation/cri-pod-container-metrics.md]: It was a goal from the KEP, but I agree it's safer to remove it.

**Comment by tzneal** [comment]
@haircommander Can you review for correctness?

**Comment by haircommander** [comment]
/lgtm

thanks @tzneal, @bobbypage PTAL

**Comment by bobbypage** [comment]
thank you for writing this up!

/lgtm

**Pull request merged**